packages/babel-plugin/src/utils/manipulate-template-literal.ts

```diff
@@ -133,9 +133,10 @@ export const moveCssPropertyInExpression = (
   // Remove CSS property so it doesn't get processed again
   if (deleteValue) {
     let nextQuasisNewCSS = nextQuasisExtractedCss.css;
+    const newValue = value.substring(0, value.lastIndexOf(before));
```

Comment:
Does this also work if the expression is before the duplicate?
i.e.
```
font-weight: ${(props) => (props.isBolded ? 'bold' : props.isPrimary ? 'normal' : 'light')};
font-weight: initial;
```

Reply:
Hey @JakeLane thanks for adding a story for it! :)

@najjiatlassian yes it does work with the above case.` font-weight: initial;` should be the nextQuasis ( the next template element); hence, it is not affected by this fix

Reply:
I see, just trying to make sure CSS order doesn't get changed.